### PR TITLE
Page review

### PR DESCRIPTION
**Pull request by kazisiddiqi**
Hello @petros ,

Hope this finds you well. I've added Warning as a new page along with hint as a description in the existing file SUMMARY.md. Should I replicate this on the read.me file? 

Thanks

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
 # Table of contents
 
 * [Introduction](README.md)
+
+* [Warning](SUMMARY.md)
+
+{% hint style="warning" %}
+**A Hint** I am a hint.
+{% endhint %}
```
Comment (petros) [inline SUMMARY.md:9]: This part here, needs to be in a new markdown file. Not in `SUMMARY.md`

File: SUMMARY.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
 # Table of contents
 
 * [Introduction](README.md)
+
```
Comment (petros) [inline SUMMARY.md:4]: You don't need the empty line here.

**Review by petros — CHANGES_REQUESTED** [review]
I like that you name your topic branch `new-page-creation` because it shows the intention and the purpose of the work.

However, `test_git_sync` for the default branch of a repository reveals you don't have extensive experience working with git repositories.

Can you figure out how to rename your default branch to `main` (which is generally how most default branches are named)? Without breaking things? Can you share the way you've done that? The steps?

Also, underscores in branch names are not used often. Dashes is the general consensus.

Finally your commit message:

> Page creation and update

How would you change it after reading:

https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

**Comment by kazisiddiqi** [comment]
Hello Petros,

Thank you for your feedback. I've changed the branch name to "main" and followed this article "https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch"

I've some concerns even though it updates the pull request but not in GitBook sync details, does that mean I need to submit a change request in Gitbook to update this?

I would rephrase the commit message as: "Create page"

File: SUMMARY.md (hunk 1 above)
Comment (kazisiddiqi) [reply SUMMARY.md]: Updated

File: SUMMARY.md (hunk 2 above)
Comment (kazisiddiqi) [reply SUMMARY.md]: Removed empty line

**Review by petros — APPROVED** [review]
Thank you for making the changed @kazisiddiqi.

I have approved this PR with just one tiny comment that most of the files in a git repository are often all lowercase by convention, except a few that are significant in some way like:

- README.md
- SUMMARY.md

and more.

That said, `WARNING.md` would have been better as `warning.md`. Just a tiny remark.
